src/oas.md

```diff
@@ -2870,10 +2870,16 @@ The OpenAPI Specification allows combining and extending model definitions using
 `allOf` takes an array of object definitions that are validated _independently_ but together compose a single object.
 
 While composition offers model extensibility, it does not imply a hierarchy between the models.
-To support polymorphism, the OpenAPI Specification adds the [`discriminator`](#schema-discriminator) field.
-When used, the `discriminator` indicates the name of the property that hints which schema definition is expected to validate the structure of the model.
-As such, the `discriminator` field MUST be a required field.
-There are two ways to define the value of a discriminator for an inheriting instance.
+
+JSON Schema also provides the `anyOf` and `oneOf` keywords, which allow defining multiple schemas where at least one or exactly one of them must be valid, respectively.
+As is the case with `allOf`, the schemas are validated _independently_.
+These keywords can be used to describe polymorphism, where a single field can accept multiple types of values.
+
+The OpenAPI specification extends the JSON Schema support for polymorphism by adding the [`discriminator`](#schema-discriminator) field whose value is a [Discriminator Object](#discriminator-object).
+When used, the Discriminator Object indicates the name of the property that hints which schema of an `anyOf` or `oneOf` is expected to validate the structure of the model.
```

Comment:
```suggestion
When used, the [Discriminator Object](#discriminator-object) indicates the name of the property that hints which schema of an `anyOf` or `oneOf` is expected to validate the structure of the model.
```

Reply:
apologies for not checking the style guide first. Thanks @ralfhandl for the reminder